cmd/commands/runtime.go

```diff
@@ -1306,6 +1328,15 @@ func RunRuntimeUninstall(ctx context.Context, opts *RuntimeUninstallOptions) err
 
 	log.G(ctx).Infof("Uninstalling runtime \"%s\" - this process may take a few minutes...", opts.RuntimeName)
 
+	err = removeGitIntegrations(ctx, opts)
+	handleCliStep(reporter.UninstallStepRemoveGitIntegrations, "Removing git integrations", err, true)
+	if err != nil {
+		if !opts.Force {
+			summaryArr = append(summaryArr, summaryLog{"you can attempt to uninstall again with the \"--force\" flag", Info})
```

Comment:
the force flag does not affect here so you should not print it here. But actually I think it would be better if it did affect and actually skipped this `removeGitIntegrations` step.